spec/integration/admin_censor_rule_spec.rb

```diff
@@ -0,0 +1,53 @@
+# -*- encoding : utf-8 -*-
+require File.expand_path(File.dirname(__FILE__) + '/../spec_helper')
+require File.expand_path(File.dirname(__FILE__) + '/alaveteli_dsl')
+
+describe 'Adding an authority censor rule' do
+  before do
+    allow(AlaveteliConfiguration).to receive(:skip_admin_auth).and_return(false)
+
+    confirm(:admin_user)
+    @admin = login(:admin_user)
+    @authority = FactoryGirl.create(:public_body)
+  end
+
+  it 'clears the cache for existing requests' do
+    raw_email_data = <<-EOF.strip_heredoc
```

Comment:
Aha - great minds! sorry, I was taking this commit by commit so I didn't see this.

It might be clearer if you moved the integration specs out of this and pushed them as a fixup to the previous commit?  

I'd also still opt for more use of `let` :)